packaging/docker/alpine-linux/Dockerfile

```diff
@@ -24,8 +24,10 @@ RUN mkdir -p /buildkite/builds /buildkite/hooks /buildkite/plugins \
     && curl -Lfs -o /usr/local/bin/ssh-env-config.sh https://raw.githubusercontent.com/buildkite/docker-ssh-env-config/master/ssh-env-config.sh \
     && chmod +x /usr/local/bin/ssh-env-config.sh
 
+ARG TARGETARCH
```

Comment:
This change might need some thinking about how to properly upstream it, it looks like the buildkite/agent image isn't multi-arch? Maybe we split this out